apis/v1alpha1/httproute_types.go

```diff
@@ -335,8 +335,10 @@ type HTTPRouteFilter struct {
 
 	// Filter-specific configuration definitions for core and extended filters
 
+	// +optional
 	RequestHeader *HTTPRequestHeaderFilter `json:"requestHeader"`
```

Comment:
Does this also need `omitempty` here?

Reply:
Probably so. I wasn't sure how nil was handled - it will output `null` without omitempty (https://play.golang.org/p/Ag3r0tr1-ox)